.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,31 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: production
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+          cache: yarn
+      - run: yarn install --frozen-lockfile
+      - run: yarn run compile
+      - run: yarn run prepare-for-extension-publish
```

Comment:
Note this step, where we update the extension version just locally on CI machine before publication